Documentation/operations/troubleshooting.rst

```diff
@@ -537,6 +537,93 @@ Opening the ``status``, we can drill down through ``policy.realized.l4``. Do
 your ``ingress`` and ``egress`` rules match what you expect? If not, the
 reference to the errant rules can be found in the ``derived-from-rules`` node.
 
+Policymap pressure and overflow
+-------------------------------
+
+The most important step is finding out which node is experiencing policymap
+pressure.
+
+The ``cilium_bpf_map_pressure{map_name="cilium_policy_*"}`` metric monitors the
```

Comment:
```suggestion
The ``cilium_bpf_map_pressure{map_name=~"cilium_policy_.*"}`` metric monitors the
```
Feel free to reject, but regex matching should be required for this wildcard to work.

Reply:
I've written these docs with https://github.com/cilium/cilium/pull/27367 in mind :)

Reply:
Oh, slick.  :smile:  Carry on, then!